lib/klaviyo_api/resources/list.rb

```diff
@@ -12,5 +12,32 @@ def find_single(scope, options)
         super.tap { |record| record.id = scope }
       end
     end
+
+    # Gets all Members of this List. Uses the Groups endpoint.
+    # Returns an enumerator that knows how to transparently deal
+    # with Klaviyo's `marker` for pagination. Pages seem to be
+    # 1000 items.
+    #
+    # https://www.klaviyo.com/docs/api/v2/lists#get-members-all
+    def members(options = {})
+      CachingEnumerator.new do |yielder|
+        marker = nil
+
+        loop do
```

Comment:
A little concerned about hitting a rate limit and what's going to happen

Reply:
I echo the concern without providing a solution

Reply:
Hmm. Yes I see your point. I also don't have a solution. The question I pose to you is - is it the gem's responsibility to enforce rate limiting? If it's used as an enumerator with `next`, then users can slow their roll. If they do something like `count` then they'll get rate limited and know not to do that...

Reply:
The gem shouldn't be mitigating the rate limit but it has to provide the user the ability to do so in between calls